app/utils/error.interceptor.js

```diff
@@ -6,11 +6,23 @@
         .module('app')
         .factory('ErrorInterceptor', ErrorInterceptor);
 
-    ErrorInterceptor.$inject = ['$q', '$injector', 'growl'];
-    function ErrorInterceptor($q, $injector, growl) {
+    ErrorInterceptor.$inject = ['$q', 'growl', '$location'];
+    function ErrorInterceptor($q, growl, $location) {
         let checkError = function (response) {
+
+            if (response && response.status == 502 || response && response.status == 504) {
```

Comment:
Please, replace to `if (response && (response.status == 502 || response.status == 504)) {`